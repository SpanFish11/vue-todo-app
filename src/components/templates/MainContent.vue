<script setup>
import ContentHeader from '@/components/molecules/ContentHeader.vue'
import AddTodo from '@/components/molecules/AddTodo.vue'
import TodoList from '@/components/organisms/TodoList.vue'
import TodoActions from '@/components/molecules/TodoActions.vue'
</script>

<template>
  <div>
    <ContentHeader />
    <AddTodo />
    <TodoList />
    <TodoActions />
  </div>
</template>
