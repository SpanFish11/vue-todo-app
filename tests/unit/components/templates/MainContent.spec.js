import { beforeEach, describe, expect, it } from '@jest/globals'
import { shallowMount } from '@vue/test-utils'
import MainContent from '@/components/templates/MainContent.vue'

describe('Given the MainContent component', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(MainContent, {})
  })

  it('should render', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
