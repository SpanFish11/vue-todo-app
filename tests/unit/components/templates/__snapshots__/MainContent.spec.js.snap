// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the MainContent component should render 1`] = `
<div
  class="container"
>
  <content-header-stub />
  <add-todo-stub />
  <todo-card-stub />
</div>
`;
